google/cloud/spanner_dbapi/cursor.py

```diff
@@ -305,14 +305,19 @@ def fetchone(self):
         self._raise_if_closed()
 
         try:
-            return next(self)
+            res = next(self)
+            self._checksum.consume_result(res)
+            return res
         except StopIteration:
             return None
 
     def fetchall(self):
         self._raise_if_closed()
 
-        return list(self.__iter__())
+        res = list(self.__iter__())
```

Comment:
Here also: Not directly relevant for this change, but this call can also cause an `Aborted` error that should initiate a transaction retry.